client/templates/calendar/calendar.html

```diff
@@ -9,30 +9,31 @@
                 </div>
             </form>
         </div>
+        {{#if Template.subscriptionsReady}}
         <div class="row">
             <h2>Upcoming gathers</h2>
         </div>
         <div class="row">
-            {{#if notEmpty upcomingEvents}}
+            <!-- {{#if upcomingEvents}} -->
                 <ul class="collapsible popout">
                 {{#each upcomingEvents}}
                     {{> calendarEvent}}
                 {{/each}}
                 </ul>
-            {{else}}
+            <!-- {{else}} -->
```

Comment:
How does it work? If conditional is commented how does `No upcoming events :( Please add one!` not get rendered?